### ISSUE TITLE
Bug: Board archives don't respect card sort orders

### ISSUE THREAD

**Issue by wuwinson** [mattermost/focalboard#2335]
## Steps to reproduce the behavior

1. Select Welcome Board from template picker
2. Rearrange cards under "TO DO" column to:
  - Create a new card
  - Manage tasks with cards
  - Create your own board
3. Export board archive from the "..." menu button
4. Go to another workspace and import the board

## Expected behavior

Cards from the imported board are sorted in alphabetical order and not the order from step 2 above.

## Screenshots
N/A

## Edition and Platform

 - Edition: Mattermost plugin
 - Version: v0.14
 - Browser and OS: Chrome 96.0.4664.55 on macOS

## Additional context
Sev/2 since this impacts the new Onboarding experience for growth. PRs need to be merged by this Thursday 2/17.

**Comment by wuwinson** [comment]
Related to https://github.com/mattermost/focalboard/issues/2311. @wiggin77 This one impacts the onboarding experience for growth. PRs need to be merged by Thursday. cc @chenilim @sbishel

**Comment by wiggin77** [comment]
> This one impacts the onboarding experience for growth. PRs need to be merged by Thursday. 

Understood.

**Comment by wiggin77** [comment]
@wuwinson does this work as expected when duplicating a board?

**Comment by wuwinson** [comment]
@wiggin77 Nope, just tried and I'm running to the exact same issue when duplicating the board as well.

**Comment by wiggin77** [comment]
That's what I suspected.  It's not a bug around import/export - this has been broken ever since the work around unique ids.
